config.yml

```diff
@@ -7,22 +7,22 @@ config:
   searchBucket: janelia-neuronbridge-search-${opt:stage, 'dev'}
   downloadBucket: janelia-neuronbridge-downloads-${opt:stage, 'dev'}
   dataBucket: janelia-neuronbridge-data-${env:SEARCH_DATA_LEVEL, opt:stage, 'dev'}
-  perDaySearchLimits: ${env:PER_DAY_SEARCH_LIMITS, 100}
-  perDayAlignmentLimits: ${env:PER_DAY_ALIGNMENT_LIMITS, 10}
+  perDaySearchLimits: ${env:PER_DAY_SEARCH_LIMITS, 50}
```

Comment:
We need to come up with some reasonable defaults for these limits

Reply:
I thought we were limiting concurrent alignments and searches, not per day? It would be ideal if users could only run one alignment and one custom search at a time. If they want to do that all day, that's fine.